web/packages/teleport/src/UnifiedResources/UnifiedResources.tsx

```diff
@@ -62,6 +65,7 @@ export function UnifiedResources() {
         key={clusterId} // when the current cluster changes, remount the component
         clusterId={clusterId}
         isLeafCluster={isLeafCluster}
+        availabilityFilterOptions={{ hidden: true, defaultMode: 'accessible' }}
```

Comment:
another thing that was confusing me is, if `hidden`, why do we care about the mode? (make it optional so we don't need to supply it?)

Reply:
the two places it is 'hidden' is the OSS (accessible) and access requests (requestable), so i figured i had to pass that value in somewhere